src/StatusBarMenu.ts

```diff
@@ -366,7 +341,7 @@ export class StatusBarMenu {
         try {
             this._isLoading = true;
             this.refreshStatusItem();
-            const kubectlClient = await this._binariesUtility.tryGetKubectlAsync();
+            var kubectlClient;
```

Comment:
💡 Same like mentioned above, `var` gets declared and then we do null value check instantly which makes this logic little flawed.